test/clj_kondo/analysis_test.clj

```diff
@@ -126,14 +126,34 @@
     (testing "no namespace for key :a"
       (let [a (analyze "#:xml{:_/a 1}"
                        {:config {:output {:analysis {:keywords true}}}})]
-        (is (= '{:row 1, :col 7, :end-row 1, :end-col 11, :name "a", :filename "<stdin>"}
-               (first (:keywords a))))))
+        (assert-submaps
```

Comment:
Can you please revert this assert-submaps change? I consciously did not use it here since the namespace should be absent and not be an underscore.

Reply:
Hum, got it, will change

Reply:
Maybe also add a comment about it why we didn't use assert-submaps there.